slices/libtbb2.yaml

```diff
@@ -0,0 +1,19 @@
+package: libtbb2
+
+essential:
+  - libtbb2_copyright
+
+slices:
+  libs:
+    essential:
+      - libatomic1_libs
```

Comment:
This is not needed, right?

```
ldd libtbb2/usr/lib/x86_64-linux-gnu/libtbb.so.2
        linux-vdso.so.1 (0x00007ffdc8cd3000)
        libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x0000780e4c002000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x0000780e4bfe2000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x0000780e4bdb9000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x0000780e4bcd2000)
        /lib64/ld-linux-x86-64.so.2 (0x0000780e4c278000)
```

Reply:
My bad, we do need `libatomic1_libs`, but for riscv64 architecture only. Thanks for pasting the image, I actually did not check the packages.ubuntu.com page!

Unfortunately, we do not yet have the functionality to specify per-architeture dependencies in Chisel. So far, what we do for scenarios like this is we leave the dependency **commented**, with an additional comment on top saying "Oh we only need it for riscv64, please install it manually..".

So here, the changes would be something like:

```yaml
slices:
  libs:
    essential:
      # libatomic1 is only required for the riscv64 architecture.
      # - libatomic1_libs
      ...
```

Regarding copyrights, it's not the case here anymore but if we don't install a package we don't install its copyright. Since we are commenting the `libatomic1_libs` bits here, we don't need to include its copyright in `essential`.

Reply:
Done, I think we've covered all comments from this thread, setting it to resolved.